paseos/tests/aero_disturbance_test.py

```diff
@@ -0,0 +1,122 @@
+import sys
+
+sys.path.append("../..")
+
+from paseos.attitude.disturbance_torques_utils import compute_aerodynamic_torque
+
+import trimesh
+import numpy as np
+
+
+#  create mesh
+vertices = [
+    [-0.5, -0.5, -0.5],
+    [-0.5, -0.5, 0.5],
+    [-0.5, 0.5, -0.5],
+    [-0.5, 0.5, 0.5],
+    [0.5, -0.5, -0.5],
+    [0.5, -0.5, 0.5],
+    [0.5, 0.5, -0.5],
+    [0.5, 0.5, 0.5],
+]
+faces = [
+    [0, 1, 3],
+    [0, 3, 2],
+    [0, 2, 6],
+    [0, 6, 4],
+    [1, 5, 3],
+    [3, 5, 7],
+    [2, 3, 7],
+    [2, 7, 6],
+    [4, 6, 7],
+    [4, 7, 5],
+    [0, 4, 1],
+    [1, 4, 5],
+]
+mesh = trimesh.Trimesh(vertices, faces)
+spacecraft_temperature_K = 300
+
+
+def test_1():
```

Comment:
please give descriptive names to tests matching what is tested, otherwise when they fail it will just say "test_1 failed" and you don't know what it does